make_prg/io_utils.py

```diff
@@ -152,13 +152,17 @@ def write_prg(prg_fname: Path, prg_string: str, options: ArgumentParser):
     Writes th prg to `output_file`.
     Writes it as a human readable string, and also as an integer vector
     """
-    with prg_fname.open("w") as prg:
-        header = f">{options.prg_name} max_nest={options.max_nesting} min_match={options.min_match_length}"
-        print(f"{header}\n{prg_string}", file=prg)
+    seqid = options.seqid or options.prg_name
```

Comment:
I like this, if I understand it uses `or` shortcircuiting to give priority to seqid :+1:

Reply:
Correct.